### ISSUE TITLE
In certain situations Time.parse will fail while interrogating process start time 

### ISSUE THREAD

**Issue by pu239ppy** [bloomberg/chef-bach#753]
https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/libraries/utils.rb#L375

Depending in how long the process has been running different ps format flags provide different output

````
ubuntu@dob2-bach-r4n04:~$ ps --noheader -o start_time 46342
 2016
ubuntu@dob2-bach-r4n04:~$ ps --noheader -o bsdstart 46342
Sep 21
ubuntu@dob2-bach-r4n04:~$ ps --noheader -o start 46342
  Sep 21
ubuntu@dob2-bach-r4n04:~$ ps --noheader -o stime 46342
 2016
ubuntu@dob2-bach-r4n04:~$ ps --noheader -o lstart 46342
Wed Sep 21 11:44:26 2016
````
I have not tested them all, but I know for sure Time.parse will choke on 2016.  This is due to how the start flag treats long running processes. Ideally this should never happen, as chef runs all the time and failures would be processed with no delay, however in one particular case, another issue was blocking chef from processing the ruby block in question.  My proposal would be to use lstart, which will not only be possible to parse with Time.parse but will also provide an absolute time for comparison as opposed to just hh:mm

**Comment by cbaenziger** [comment]
Erg, this has happened before. We should use `-lstart` is what we found in the past.

**Comment by pu239ppy** [comment]
Yep that's what I said above, I will put in a PR tomorrow

**Comment by vt0r** [comment]
Should be fixed by #754
